### PR TITLE
feat: vue-loader@15.x support experimental inline match resource

### LINKED ISSUE #1: Support vue-loader@15 with Vue@2.x
Basically, this features will add support for vue-loader@15 for Vue@2.x.

1. Support `experiments.css` auto-detection
2. Support builtin `type: "ts"`

You can take this PR for vue-loader@16 as a reference: https://github.com/vuejs/vue-loader/pull/2046

### PR DESCRIPTION
**Pull request by jeromehan**
@h-a-n-a  Maybe fix #1

### CODE REVIEW

File: lib/loaders/pitcher.js [hunk 1]
```diff
@@ -111,15 +123,46 @@ module.exports.pitch = function (remainingRequest) {
...
+        )
+        return ''
+      }
+
+      const loaderString = [stylePostLoaderPath, ...loaders]
+        .map((loader) => {
+          return typeof loader === 'string' ? loader : loader.request
+        })
+        .join('!')
+      return `@import "${this.resourcePath}${
```
Comment (h-a-n-a) [inline lib/loaders/pitcher.js]: This one is missing https://github.com/vuejs/vue-loader/pull/2049/files#diff-0215db87c4960efcf6d1f7885aedb12aacb5ed6b4aefde4b1fde67849beefa34R91

Comment (jeromehan) [reply lib/loaders/pitcher.js]: ok

**Comment by h-a-n-a** [comment]
Thank you!

**Review by h-a-n-a — APPROVED** [review]

**Comment by hardfist** [comment]
thank you, @jeromehan are you willing to backport this to vue-loader repo, @sodatea seems willing to accept this change

**Comment by jeromehan** [comment]
@hardfist  Already in Vue-Loader PR #2058, don't know when there will be merged

**Pull request merged**